test/functional/framework.go

```diff
@@ -470,6 +486,27 @@ func (f *FluentdFunctionalFramework) ReadApplicationLogsFrom(outputName string)
 	return f.ReadLogsFrom(outputName, applicationLog)
 }
 
+func (f *FluentdFunctionalFramework) ReadApplicationLogsFromKafka(topic string, brokerlistener string, consumercontainername string) (results []string, err error) {
+	//inter broker zookeeper connect is plaintext so use plaintext port to check on sent messages from kafka producer ie. fluent-kafka plugin
```

Comment:
I see that you are reading from a file "/shared/consumed.logs" in this function. If this is the case, you can use the function "ReadLogsFrom" and you do not need to reimplement it. You need to add the needed values to outputLogFile and than this function will work like all the rest of the wrappers.